modules/installation-creating-aws-worker.adoc

```diff
@@ -0,0 +1,166 @@
+create worker nodes in Amazon Web Services (AWS) for your cluster to use.
+The easiest way to manually create these nodes is to modify the provided
+CloudFormation template.
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your worker
+nodes, you must review the provided information and manually create
+the infrastructure. You might need to consult Red Hat support to ensure that
+your cluster initializes correctly.
+====
+
+[WARNING]
+====
+It is better to use a MachineSet to allow the cluster to create and manage
+worker nodes than to manually create them.
+====
+
+.Prerequisites
+
+* Configure an AWS account.
+* Generate the ignition config files for your cluster.
+* Create and configure a VPC and assocated subnets in AWS.
+* Create and configure DNS, load balancers, and listeners in AWS.
+* Create control plane and worker roles.
+* Create the bootstrap node.
+* Create the control plane nodes.
+
+.Procedure
+
+. Upload the worker ignition config file to the S3 bucket that you created for
+your bootstrap node.
+See link:https://docs.aws.amazon.com/AmazonS3/latest/user-guide/create-bucket.html[How Do I Create an S3 Bucket?]
+in the AWS documentation.
+
+. Create a YAML file that contains a copy of the CloudFormation template that
+describes the networking objects and load balancers and save it to your computer.
+
+. Create a JSON file that contains the parameter values that the template
+requires:
++
+[source,json]
+----
+[
+  {
+    "ParameterKey": "ClusterName", <1>
+    "ParameterValue": "mycluster" <2>
+  },
+  {
+    "ParameterKey": "RhcosAmi", <3>
+    "ParameterValue": "ami-07f604c5f18a1e7a9" <4>
+  },
+  {
+    "ParameterKey": "WorkerSubnet", <5>
+    "ParameterValue": "yes" <6>
+  },
+  {
+    "ParameterKey": "WorkerSecurityGroupId", <7>
+    "ParameterValue": "sg-042ebc4dd65c71d67" <8>
+  },
+  {
+    "ParameterKey": "IgnitionLocation", <9>
+    "ParameterValue": "https://api.<cluster_name>.<domain_name>:22623/config/master" <10>
+  },
+  {
+    "ParameterKey": "CertificateAuthorities", <11>
+    "ParameterValue": "" <12>
+  },
+  {
+    "ParameterKey": "WorkerInstanceProfileName", <13>
+    "ParameterValue": "" <14>
+  },
+  {
+    "ParameterKey": "WorkerInstanceType", <15>
+    "ParameterValue": "m4.large" <16>
+  },
+  {
+    "ParameterKey": "AutoRegisterELB", <27>
+    "ParameterValue": "yes" <28>
+  },
+  {
+    "ParameterKey": "RegisterNlbIpTargetsLambdaArn", <29>
+    "ParameterValue": "1" <30>
+  },
+  {
+    "ParameterKey": "ExternalApiTargetGroupArn", <31>
+    "ParameterValue": "1" <32>
+  },
+  {
+    "ParameterKey": "InternalApiTargetGroupArn", <33>
+    "ParameterValue": "1" <34>
+  },
+  {
+    "ParameterKey": "InternalServiceTargetGroupArn", <35>
+    "ParameterValue": "1" <36>
+  }
+]
+----
+<1> A short, representative cluster name to use for hostnames, etc.
+<2> Specify the cluster name that you used when you generated the
+`install-config.yaml` file.
```

Comment:
cluster name is also used when user decide to generate only ignition-configs files.